doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -209,70 +209,99 @@ infrastructure itself from the workloads compute resources.
+via CNI integration.
+- **Service Mesh**: The well known service meshes are "application service meshes" that address and
+interact with the application layer 7 protocols (eg.: HTTP) only. Therefore,
+their support is not required in this architecture, as these service meshes are
+outside the scope of the infrastructure layer of the CNTT stack.
 
-A CNI metaplugin/CNI multiplexer has the capability to attach several
-interfaces, using different other CNI plugins, to a pod. Note that the different
-network characteristics of the interfaces might require different networking
-technologies, which would potentially require different CNI plugins.
+<p align="center"><img src="../figures/ch03_networking.png" alt="Kubernetes Networking Architecture" Title="Kubernetes Networking Architecture" width="100%"/></p>
+<p align="center"><b>Figure 3-2:</b> Kubernetes Networking Architecture</p>
 
-To comply with `req.inf.ntw.08`, inter node communication must be served by a
-CNI plugin which complies with the default [Kubernetes networking assumptions](https://kubernetes.io/docs/concepts/cluster-administration/networking/#the-kubernetes-network-model).
+<!--The above diagram is maintained here: https://wiki.lfnetworking.org/display/LN/CNTT+RA2+-+Kubernetes+-+Diagrams+-+Networking-->
 
 There are two types of low latency and high throughput networks required by
 telco workloads: signalling traffic workloads and user plane traffic workloads.
 Networks used for signalling traffic are more demanding than what a standard
 overlay network can handle, but still do not need the use of user space
 networking. Due to the nature of the signalling protocols used, these type of
-networks require NAT-less communication documented in infra.net.cfg.003 and will
+networks require NAT-less communication documented in `infra.net.cfg.003` and will
 need to be served by a CNI plugin with IPVLAN or MACVLAN support. On the other
 hand, the low latency, high throughput networks used for handling the user plane
 traffic require the capability to use a user space networking technology.
 
-> Note: An infrastructure can provide the possibility to use SR-IOV with DPDK as
-an additional feature and still be conformant with CNTT.
-
-> Editors note: The possibility to SR-IOV for DPDK is under discussion.
-
-The integration of SDN solutions required by `req.inf.ntw.05` should be enabled
-via their respective CNI integration.
 
-> Note: An SDN solution can manage the pod networks via the Kubernetes API or
-can have a CNI integration.
+| Requirement | Networking Solution with Multus | Networking Solution with DANM |
+|---|---|---|
+| The overlay network encapsulation protocol needs to enable ECMP in the underlay (`infra.net.cfg.002`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin |
+| NAT (`infra.net.cfg.003`) | Supported via the additional CNI plugin | Supported |
+| Security Groups (`infra.net.cfg.004`) | Not supported | Not supported <sub>(1)<sub> |
```

Comment:
Do we need Security Groups (an AWS term) or can the same be accomplished through Security Policies? May need to change RM and other places in RA2

Reply:
Yes, I think we mean Kubernetes Network Policies here.

Reply:
Updated in https://github.com/cntt-n/CNTT/pull/2121/commits/0ef603d4c08dd9c0b3b86034353ebd9de31d826a.  @pgoyal01 ok to resolve?

Reply:
Yes -- it is resolved